PowerEditor/src/WinControls/Preference/preference.rc

```diff
@@ -130,6 +130,7 @@ BEGIN
     CONTROL         "Enable Notepad++ auto-updater",IDC_CHECK_AUTOUPDATE, "Button",BS_AUTOCHECKBOX | BS_MULTILINE | WS_TABSTOP,37,173,217,10
     CONTROL         "Enable",IDC_CHECK_CLICKABLELINK_ENABLE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,267,15,140,10
     CONTROL         "No underline",IDC_CHECK_CLICKABLELINK_NOUNDERLINE, "Button",BS_AUTOCHECKBOX | WS_TABSTOP,267,28,140,10
+	CONTROL         "Always sort function list",IDC_FUNCTION_LIST,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,267,110,140,10

```

Comment:
@Kataderon:  FYI, this line is indented with a tab, but the lines above and below are indented with spaces.

Reply:
Fixed